File: 05_arrays/05_01_find_largest/find_largest.cpp
#include<bits/stdc++.h>

using namespace std;

int findMax(int arr[], int n) {
    int max = arr[0];
    for (size_t i = 1; i < n; i++)
    {
        if (max < arr[i])
        {
            max = arr[i];
        }
    }
    
    return max;
}

int main() {
    int arr[] = { 2, 5, 1, 3, 0 };
    int n = 5;

    cout << "Elements:- " << endl;
    for (size_t i = 0; i < n; i++)
    {
        cout << arr[i] << " ";
    }
    cout << endl;
    
    cout << "Max: " << findMax(arr, n) << endl;
}

File: 05_arrays/05_01_find_largest/find_largest.js
const findMax = (arr = []) => {
    let max = arr[0];

    for (let index = 0; index < arr.length; index++) {
        if (arr[index] > max) {
            max = arr[index];
        }
    }

    return max;
}

const array = [2, 3, 5, 1, 0];
console.log("Elements:-");
console.log(array);
console.log("Max:-");
console.log(findMax(array));
